src/Features/Core/Portable/Snippets/SnippetProviders/AbstractInlineStatementSnippetProvider.cs

```diff
@@ -24,7 +24,8 @@ internal abstract class AbstractInlineStatementSnippetProvider : AbstractStateme
         /// Tells if accessing type of a member access expression is valid for that snippet
         /// </summary>
         /// <param name="type">Type of right-hand side of a member access expression</param>
-        protected abstract bool IsValidAccessingType(ITypeSymbol type);
+        /// <param name="compilation">Current compilation instance</param>
+        protected abstract bool IsValidAccessingType(ITypeSymbol type, Compilation compilation);
```

Comment:
This change makes a lot of sense since individual snippets might want to compare type to other types in compilation. It just happened that all previously implemented snippets where checked using their special type